amundsen_application/static/webpack.dev.ts

```diff
@@ -1,4 +1,27 @@
+import * as path from 'path';
 import merge from 'webpack-merge';
-import commonConfig from './webpack.common'
+import commonConfig, {htmlWebpackPluginConfig} from './webpack.common';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 
-export default merge(commonConfig, {});
+export default merge(commonConfig, {
+  devServer: {
+      contentBase: path.join(__dirname, '../build/lib/amundsen_application/static/dist'),
```

Comment:
Where does the path `'../build/lib/amundsen_application/static/dist'` come from and what behavior does it contribute to? I'm looking to understand the difference between this and the `publicPath`.

Reply:
`contentBase` is where to serve content from, and this is the output content, including static files that seems to be in the repo. `publicPath` is what the prefix will be from the port. 
https://webpack.js.org/configuration/dev-server/#devserverpublicpath-
https://webpack.js.org/configuration/dev-server/#devservercontentbase

Reading through the documentation again, theoretically this could be just 
`contentBase: ../build/lib/amundsen_application` and omit the public path, but I'd need to play around with it.